### PR TITLE
Pull Request for Tanzu Packages Installation Doc

### PR DESCRIPTION
**Pull request by manishjha86**

### CODE REVIEW

**Review by MarilynHorn — CHANGES_REQUESTED** [review]
Completed IX review of:
- tko-package-install.md
- tko-on-vsphere-with-tanzu
- index.md (minor edit)

File: src/reference-designs/tko-on-vsphere-with-tanzu.md [hunk 1]
```diff
@@ -20,82 +56,132 @@ This document lays out a reference design for deploying VMware Tanzu for Kuberne
...
 
 **Note:** vSAN File Service is available only in vSAN Enterprise and Enterprise Plus editions.
 
+## **Tanzu Kubernetes Clusters Networking**
+
+A Tanzu Kubernetes cluster provisioned by the Tanzu Kubernetes Grid supports two Container Network Interface (CNI) options:
+
+- [Antrea](https://antrea.io/)
+- [Calico](https://www.tigera.io/project-calico/)
+
```
[the remark this answers is not in the document]
Comment (manishjha86) [reply src/reference-designs/tko-on-vsphere-with-tanzu.md:159]: Multus CNI provides pod networking only and is dependent on Antrea or calico. The section where antrea and calico is discussed is for tanzu Kubernetes cluster networking and is different than pod networking

File: src/reference-designs/tko-on-vsphere-with-tanzu.md [hunk 2]
```diff
@@ -20,82 +56,132 @@ This document lays out a reference design for deploying VMware Tanzu for Kuberne
...
+Both are open-source software that provides networking for cluster pods, services, and ingress.
+
+When you deploy a Tanzu Kubernetes cluster using Tanzu CLI using the default configuration, Antrea CNI is automatically enabled in the cluster.
```
[the remark this answers is not in the document]
Comment (manishjha86) [reply src/reference-designs/tko-on-vsphere-with-tanzu.md:162]: > 

The right statement should be as follows:

When you deploy a Tanzu Kubernetes cluster using the default configuration by invoking Tanzu CLI, Antrea CNI is automatically enabled in the cluster.

**Comment by manishjha86** [comment]
Deleting this pull request as it has unwanted commits from other pull requests that was raised accidentally from my forked main repo

**Pull request closed without merging**